application/backend/app/models.py

```diff
@@ -162,27 +167,17 @@ class FacilityType(SQLModel, table=True):
     description: str | None = None
     icon_image_url: str | None = None
 
-    # Relationships
-    user_facilities: "UserFacility" = Relationship(
+    recipes: list["Recipe"] = Relationship(
         back_populates="facility_type", cascade_delete=True
     )
 
 
-class UserFacility(SQLModel, table=True):
-    id: int | None = Field(default=None, primary_key=True)
-    user_id: uuid.UUID = Field(foreign_key="user.id", ondelete="CASCADE")
-    facility_type_id: int = Field(foreign_key="facilitytype.id", ondelete="CASCADE")
-    quantity: int = Field(default=0)
-
-    # Relationships
-    user: "User" = Relationship(back_populates="facilities")
-    facility_type: "FacilityType" = Relationship(back_populates="user_facilities")
-
-
 class Recipe(SQLModel, table=True):
     id: int | None = Field(default=None, primary_key=True)
```

Comment:
Why nullable?

Reply:
The ID fields are not truly nullable. If a null value is passed in, then it auto generates an ID